ChopChop/Views/Cooking/CountdownTimerView.swift

```diff
@@ -2,76 +2,122 @@ import SwiftUI
 
 struct CountdownTimerView: View {
     @ObservedObject var viewModel: CountdownTimerViewModel
+    @State var opacity = true
```

Comment:
Should probably follow naming convention, maybe like `isOpaque`

Reply:
Also, to separate concerns, maybe this could be inside the view model instead as a `@Published var isOpaque`.

Reply:
It's in the view instead of the viewmodel because it's purely a view state (i.e. only the view ever cares about this state). `isOpaque` also isn't quite accurate since it doesn't actually correspond to whether the view is opaque or not. It's just a hacky way to make things blink in SwiftUI.